mtp_cashbook/templates/cashbook/includes/nomis_print_dialog.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+{% load staticfiles %}
+
+<div id="print-dialog" class="Dialog" style="display: none">
+  <div class="Dialog-inner mtp-change-notification__dialog">
+    <h3 class="heading-medium">{% trans "Planning to enter these credits by hand?" %}</h3>
+    <img class="mtp-change-notification__graphic" src="{% static 'images/nomis-change-graphic.png' %}"/>
+    <img class="mtp-change-notification__graphic2x" src="{% static 'images/nomis-change-graphic@2x.png' %}"/>
```

Comment:
would be simpler to use a `srcset` attribute on one image element and avoid having to do any styles. like in the training section